### PR TITLE
More LCSC codes

### PR DESCRIPTION
**Pull request by rfvermut**

### CODE REVIEW

File: Hardware/OpenMowerMainboard/RobotConnectors.kicad_sch [hunk 1]
```diff
@@ -3287,6 +3290,12 @@
     (property "Stock_PN" "FH-FC-202-24" (id 6) (at 118.745 68.58 0)
       (effects (font (size 1.27 1.27)) hide)
     )
+    (property "Description" "LCSC links to resettable PTC" (id 7) (at 118.745 68.58 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "LCSC" "C369104" (id 8) (at 118.745 68.58 0)
```
Comment (ClemensElflein) [inline Hardware/OpenMowerMainboard/RobotConnectors.kicad_sch]: Not sure if that is the right fuse to use. The fuse trips at 10A for 11 seconds. That's about 300W of power for 11 seconds, something is going to get hot.

**Review by ClemensElflein — COMMENTED** [review]
Looks good to me except for the fuse. I'm personally not a fan of these slow, resetable fuses. I can't see the benefit in this case, if any component gets 300W for 11sec you'll need to open the mower to repair anyways; Opening to "just" replace the fuse makes more sense to me.

File: Hardware/OpenMowerMainboard/RobotConnectors.kicad_sch (hunk 1 above)
Comment (rfvermut) [reply Hardware/OpenMowerMainboard/RobotConnectors.kicad_sch]: I'll find something faster in catalogue.

**Comment by rfvermut** [comment]
Reverted fuses back to Ali fuse holders.

Merged #24 changes in

**Comment by rfvermut** [comment]
also added 3D model path fixes to this PR, now more components are rendered
image

**Pull request merged**